tsi_app/m85/sample.yaml

```diff
@@ -0,0 +1,19 @@
+sample:
+  description: TSI Core platform  application on M85
+  name: m85
+common:
+  tags: introduction
+  integration_platforms:
+    - ek_tsi_skyp 
```

Comment:
You can check if we need to rename this file ? As sample.yml is not clear what it is intended for.